### DISCUSSION TITLE
Download failure

### DISCUSSION THREAD

**Discussion by tonyrose9999** [aclap-dev/video-downloadhelper#1178] [Help and Q&A]
### Your question

I've attached the error text.

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.wptv.com/life/good-news/you-can-pick-which-dc-comics-superheroes-you-want-to-see-on-real-us-coins",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 000002b3eaf67640] Error opening input: Invalid data found when processing input\r\nError opening input file https://assets.scrippsdigital.com/cms/video/player.html?video=https://content.uplynk.com/b823f1d6357249879c552f829737bae0.m3u8&mp4=https://x-default-stgec.uplynk.com/ause/slices/b82/e63aadb6928b4cd6ab0dec6fb3cddabd/b823f1d6357249879c552f829737bae0/b823f1d6357249879c552f829737bae0_h.mp4&autoplay=false&purl=/life/good-news/you-can-pick-which-dc-comics-superheroes-you-want-to-see-on-real-us-coins&ads.iu=/6088/ssp.wptv/life/good-news/you-can-pick-which-dc-comics-superheroes-you-want-to-see-on-real-us-coins&ads.proxy=1&poster=https://x-default-stgec.uplynk.com/ause/slices/b82/e63aadb6928b4cd6ab0dec6fb3cddabd/b823f1d6357249879c552f829737bae0/poster_1fbb0b289d9f4f2e9d665ffc13460019.jpg&title=See%20your%20favorite%20superheroes%20on%20U.S.%20coins&kw=&contplay=*recent&mute=0&cust_params=temp%3D%26weather%3D&host=wptv.com&s=wptv&env=production.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_eb0a9784-426e-472f-91bf-902b79156d00",
    "downloadable_id": "downloadable_8129667843977712"
  }
}

**Comment by Mattmjs** [comment]
Your extension version is not the latest, even so when I tried with version 9.1.0.42 (stable) I got an error on that site.
The site is ridiculous in how it behaves and what it requires to make it work. I tried playing the video but it would have a
continuous spinning circle. I thought it was the ad blocker so I disabled it, reloaded and clicked play. Still a spinning circle.
Then I thought to also disable tracking protection. Click play and bam the video plays. They have somehow made it so that
the video only works by allowing all of those trackers. Once this is done, VDH will detect the video and download it.

The important thing to note is it detects the video with tracking protection on, but the media has no duration.
With tracking protection off, it is detected with the videos duration.
